docs/resources/users_permission.md

```diff
@@ -0,0 +1,85 @@
+# keycloak_users_permissions
+
+Allows you to manage all users Scope Based Permissions.
+
+This is part of a preview keycloak feature. You need to enable this feature to be able to use this resource.
+More information about enabling the preview feature can be found here: https://www.keycloak.org/docs/latest/securing_apps/index.html#_token-exchange
```

Comment:
Is this the correct link for enabling this preview?

Reply:
nope ;)